benchmarks/vec-saxpy/vec-saxpy.S

```diff
@@ -0,0 +1,30 @@
+    .text
+    .balign 4
+    .global vec_saxpy
+# void
+# vec_saxpy(size_t n, const double a, const double *x, const double *y, double* z)
```

Comment:
This is actually DAXPY (i.e. double precision), not SAXPY.  I'd be fine with changing either correcting the name or changing the data type.

Reply:
Oops. Fixed